app/src/main/res/values/strings.xml

```diff
@@ -147,6 +147,7 @@
     <string name="create_account_activity_error_wrong_telephone_number">It\'s not possible to create a Simlar account with the number you provided:\n%1$s\n\nTry another number and contact the mailing list.</string>
     <string name="create_account_activity_error_sms">We are currently not able to send an SMS to the number you provided:\n%1$s\n\nCheck your number or try again later.</string>
     <string name="create_account_activity_error_sms_timeout">Waiting for SMS timed out.\n\nIs this really your number?\n%1$s\n\nIf you received the registration SMS, enter the code manually:</string>
+    <string name="create_account_activity_error_sms_not_granted">Permission to read sms not granted.\n\nSimlar send a sms to this number:\n%1$s\n\nOnce you received the this SMS, enter the code manually:</string>
```

Comment:
Simlar has sent an SMS to the number you provided:\n%1$s\n\nOnce you receive this SMS, enter the code manually: